guide/release-notes/v3.0.7.adoc

```diff
@@ -12,6 +12,6 @@ Release date:: 2020-11-01
 * Rule PEPPOL-COMMON-R040: "GLN must have a valid format according to GS1 rules". Changed from severity "warning" to "fatal". (The rule was introduced in fall 2019 with severity "warning" to avoid disruptions but with the intention to be changed to severity "fatal" after 6-12 months). Issue: [POACC-333] and [POACC-341]
 * EAS code list, added code 0209, removed code 9958. Rule updated accordingly.
 * ICD code list, added code 0209. Rule updated accordingly. Names and descriptions added where missing.
-* VATEX code list, added code VATEX-EU-79C for repayment of expenditure. Rule updated accordingly.
+* VATEX code list, added code VATEX-EU-79C for repayment of expenditure. Codes put in upper case.
```

Comment:
Was the VATEX code list previously used with low letter codes? This could create a backwards incompatibility

Reply:
Yes the VATEX code was previously in low letter but this was changed to align with EU Publication Office conventions. The validation rule in TS434 is however implemented as case neutral to provide backwards compatiblility. This change from lower to upper case. This was noted in the Peppol spring release by referencing the TC434 / CEF update to the EN16931 rules, https://github.com/ConnectingEurope/eInvoicing-EN16931/releases/tag/validation-1.3.1